assets/components/pages/project/curate/VariantData.js

```diff
@@ -1,6 +1,11 @@
 import PropTypes from "prop-types";
 import React from "react";
 import { Button, List } from "semantic-ui-react";
+import { Chart, registerables } from "chart.js";
+import { Bar } from "react-chartjs-2";
```

Comment:
Can you explain what Chart.register(...registerables) means here?
`chart.js` does have [bar chart functionality](https://www.chartjs.org/docs/latest/charts/bar.html), can we do this with fewer new dependencies?

Reply:
@illusional thanks, I've updated the PR to only use chart.js for now.

The `Chart.register(...registerables)` is for the scales used in the charts. The scale type in the options is set to `"linear"`, and without the `register` call the page will not display:
![Screenshot 2024-01-18 at 2 47 18 pm](https://github.com/populationgenomics/variant-curation-portal/assets/34049565/8326ee07-50ad-4f73-934d-67a39e398a96)

I may be misunderstanding the necessity of the `scales.type` option, but when I remove the `type: "linear"` line I get the same error page but with `"categorical"` in place of `"linear"` in the error (i.e `"categorical" is not a registered scale`).

Reply:
Yeah fair enough, I had no clue, could you leave a comment with this information?
It looks like it's due to charts.js being tree-shakeable (only bundles the used components).

You could probably explicitly import some functionality, but easier to register it, here's a couple of links:

- https://www.chartjs.org/docs/next/getting-started/integration.html#bundlers-webpack-rollup-etc
- https://stackoverflow.com/a/67240031